.github/workflows/main.yml

```diff
@@ -7,13 +7,17 @@ jobs:
     name: Linux x64_64
     runs-on: ubuntu-latest
 
+    env:
+      DEPLOY_GTK3: 1
+
     steps:
     - uses: actions/checkout@v2
 
     - name: Set environment variables
       run: |
         echo "REPODIR=$GITHUB_WORKSPACE" >> $GITHUB_ENV
         echo "APPDIR=/tmp/AppDir"        >> $GITHUB_ENV
+        echo "DEPLOY_GTK_VERSION=3"      >> $GITHUB_ENV
```

Comment:
Why not use the `env:` key only? This is somewhat redundant.

Reply:
You are right. Fixed in 2d9be7b8376b4a99a737092aa841ae6a5def02d7.